Microsoft.Maui.BuildTasks-net6.sln

```diff
@@ -30,6 +30,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Resizetizer.UnitTests", "sr
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{FF16050F-B905-4DE1-92FA-8B9EC93EE009}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Controls.SourceGen-net6", "src\Controls\src\SourceGen\Controls.SourceGen-net6.csproj", "{3E40598B-A52D-417D-B856-240847F6A43F}"
+EndProject
```

Comment:
why does the BuildTask.sln require SourceGen ?

Reply:
Probably not necessary but I was having issues getting the build tasks to compile at one point and just ended up adding the SourceGen project to everybody to make sure it was getting built.  It shouldn't harm anything having it added for now

Hopefully we can delete all of these solutions in a couple of months :-)